src/homework1.ts

```diff
@@ -0,0 +1,139 @@
+class School {
+    directions: string[] = [];
+  
+    addDirection(direction: string): void {
+      this.directions.push(direction);
+    }
+  }
+  
+  class Direction {
+    levels: string[] = [];
+  
+    _firstName: string;
+  
+    get firstName(): string {
+      return this._firstName;
+    }
+  
+    constructor(name: string) {
+      this._firstName = name;
+    }
+  
+    addLevel(level: string) {
```

Comment:
Please do not forget to add explicit type annotation for the return function value.